.github/workflows/github-release.yml

```diff
@@ -0,0 +1,27 @@
+# https://github.com/marketplace/actions/create-a-release
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
I totally stole this from the base Github example, but _basically_, I think the way this works is you:

- Cut a `git tag` locally
- Push up your `git tag` to Github
- Github will cut a release with the name of the tag that you cut locally